napari/components/_tests/test_viewer_model.py

```diff
@@ -1067,3 +1067,114 @@ def test_get_status_text():
     viewer.update_status_from_cursor()
     assert viewer.status == ' [1 2] » Labels: 0; a: 1    '
     assert viewer.tooltip.text == 'a: 1'
+
+
+def test_reset_view_margin():
+    """Test reset_view with different margin values."""
+    viewer = ViewerModel()
+    viewer.add_image(np.random.random((10, 10)))
+
+    # Reset view with default margin (0.05)
+    viewer.reset_view()
+    default_zoom = viewer.camera.zoom
+
+    viewer.reset_view(margin=0.2)
+    large_margin_zoom = viewer.camera.zoom
+
+    viewer.reset_view(margin=0)
+    no_margin_zoom = viewer.camera.zoom
+
+    # Check zoom decreases with increased margin
+    assert default_zoom > large_margin_zoom
```

Comment:
Style thought: I would normally put the assertion right after the code that sets large_margin_zoom and  no_margin_zoom.